bin/standalone-installer-unix

```diff
@@ -63,6 +63,21 @@ main() {
     mkdir -p "$(dirname "$DESTINATION")"
     mv standalone "$DESTINATION"
 
+    if [[ "$(type -p nextstrain 2>/dev/null)" != "$DESTINATION/nextstrain" ]]; then
+        log "Making Nextstrain CLI available for the current user"
+
+        local shell rc
+        shell="$(default-shell)"
+        rc="$(shell-rc "$shell")"
+
+        # Initialize for this session.
+        eval "$("$DESTINATION/nextstrain" init-shell $shell)"
```

Comment:
You may know this, but this won't affect the calling shell session, just the session this program is running within.